### PR TITLE
add files

### PR DESCRIPTION
**Pull request by Lopez-JhonLay**
adding my files that are needed in main branch

### CODE REVIEW

File: lopez_jhonlay/script.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+function onTextChange() {
+    let message = document.querySelector("#message").value.trim();
+    let name = document.querySelector("#name").value.trim();
+    let commentButton = document.querySelector("#comment_button");
+
+    if (message.length > 0 && name.length > 0) {
```
Comment (despe-reymondchristian) [inline lopez_jhonlay/script.js]:
```suggestion
    if (message.length && name.length) {
```

File: lopez_jhonlay/lopez_jhonlay.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                        <li>Go to different countries</li>
+                        <li>Financial Freedom</li>
+                    </ul>
+                    <h2>Comment Section</h2>
+                    <div class="comment-section">
+                        <span>
+                            <p>Nel: Nice work buddy!</p>
+                        </span>
+                        <span>
+                            <p>Pearl: Go, future dev~ Ang galing galing mo na</p>
```
Comment (despe-reymondchristian) [inline lopez_jhonlay/lopez_jhonlay.html]: exceeded 80 columns
```suggestion
                            <p>Pearl: Go, future dev~ Ang galing galing mo na</p>
```

**Review by despe-reymondchristian — CHANGES_REQUESTED** [review]
see suggested changes

File: lopez_jhonlay/script.js (hunk 1 above)
Comment (Lopez-JhonLay) [reply lopez_jhonlay/script.js]: Got it

File: lopez_jhonlay/lopez_jhonlay.html (hunk 2 above)
Comment (Lopez-JhonLay) [reply lopez_jhonlay/lopez_jhonlay.html]: Got it

**Review by despe-reymondchristian — APPROVED** [review]
Done!

**Pull request merged**